### ISSUE TITLE
Model selection

### ISSUE THREAD

**Issue by SawanKumar28** [jacobandreas/concepts#6]
Hi @jacobandreas, this is a very interesting work. I wanted to understand how you selected the model for reporting the numbers in the paper, specifically the few-shot classification task. Was it the best model on the validation set? How many epochs of training did you run?

**Comment by jacobandreas** [comment]
Everything was tuned on the validation set. You can see the parameters (num
epochs etc.) by looking at the run.sh files in the subdirectories of `exp`.

**Comment by SawanKumar28** [comment]
Thanks.
